WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -79,8 +119,10 @@ struct AddEditCoupon: View {
 
                             Group {
                                 TitleAndValueRow(title: Localization.couponExpiryDate,
-                                                 value: .placeholder(Localization.couponExpiryDatePlaceholder),
-                                                 selectionStyle: .disclosure, action: { })
+                                                 value: viewModel.expiryDateValue,
+                                                 selectionStyle: .disclosure, action: {
+                                    showingCouponExpiryActionSheet = true
```

Comment:
I'm not doing that because the SwiftUI `DatePicker` doesn't allow to pass a `nil` date. This means that when we open the view, the current date will be automatically selected, and this is not we want. 👍